imagersite/photo/models.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Photos(models.Model):
```

Comment:
In general, the "django way" is to leave the names of models singular.  It looks odd to the experienced Django programmer to see a model named `Photos`